.github/workflows/wash.yml

```diff
@@ -26,7 +20,29 @@ permissions:
   contents: read
 
 jobs:
+  should_run:
```

Comment:
I like the idea of calling this `meta` like [linkerd does in their CI](https://github.com/linkerd/linkerd2/blob/c463750c6dc497ae4acc0b9bbb2c337bea5e0efd/.github/workflows/integration.yml#L25-L51), basically the same idea as what you're introducing here.